### ISSUE TITLE
tour: Lack of description about converting integer to string

### ISSUE THREAD

**Issue by y-okamoto-1113** [golang/tour#1251]
Context: https://tour.golang.org/basics/13

It is lacking of description about converting Integer to String.
`string` method receives an argument.
If an argument is `int` or `uint`, it is recognized as a codepoint like below.
https://play.golang.org/p/x_88Grgd4ZG

And, If we want to convert `integer` to `string`, we must use `strconv.Itoa`.
Beginners may be confused.
So I think we should add the annotation.

**Comment by ALTree** [comment]
Yeah, it's a common gotcha for beginners. May be worth adding a note in that lesson.
